pkg/compute/rust.go

```diff
@@ -231,18 +231,25 @@ func (r Rust) Verify(out io.Writer) error {
 		return newCargoUpdateRemediationErr(err, latestFastly.String())
 	}
 
-	// If fastly-sys version doesn't meet our constraint of being within the
-	// minor range, error with dual remediation steps.
-	if ok := fastlySysConstraint.Check(fastlySysVersion); !ok {
-		return newCargoUpdateRemediationErr(fmt.Errorf("fastly crate not up-to-date"), latestFastly.String())
-	}
-
 	fastlyVersion, err := getCrateVersionFromMetadata(metadata, "fastly")
 	// If fastly crate not found, error with dual remediation steps.
 	if err != nil {
 		return newCargoUpdateRemediationErr(err, latestFastly.String())
 	}
 
+	// If fastly crate version is a prerelease, exit early. We assume that the
+	// user knows what they are doing and avoids any confusing messaging to
+	// "upgrade" to an older version.
+	if fastlyVersion.Prerelease() != "" {
+		return nil
+	}
+
+	// If fastly-sys version doesn't meet our constraint of being within the
+	// minor range, error with dual remediation steps.
+	if ok := fastlySysConstraint.Check(fastlySysVersion); !ok {
+		return newCargoUpdateRemediationErr(fmt.Errorf("fastly crate not up-to-date"), latestFastly.String())
```

Comment:
Is `not up-to-date` an accurate description? (based on my interpretation of the code comment above, it sounds like we're _constraining_ to a minor version) 

So should it be something like:

```suggestion
		return newCargoUpdateRemediationErr(fmt.Errorf("fastly-sys crate invalid"), latestFastly.String())
```

...although that's a bit vague 😬

Reply:
Happy to change, however this wasn't an addition in this PR, I just moved the block for control flow. Also the original message intentionally references `fastly` and not `fastly-sys` as `fastly-sys` is transitive and thus would be confusing to the user as they may not know it even exists, so we opted to tell them to upgrade their `fastly` version which in turn would upgrade `fastly-sys`. Hope that makes sense.